igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/stash/client/StashMaster.groovy

```diff
@@ -26,8 +26,10 @@ import retrofit.RetrofitError
  */
 @Slf4j
 class StashMaster extends AbstractScmMaster {
-    StashClient stashClient
-    String baseUrl
+  public static final int DEFAULT_TEXT_FILE_MAX_LINES = 2000
```

Comment:
This should be a safe limit for now, is my assumption. I'm hoping to buy time so we can circle back to handling pagination later.

Reply:
sorry to ask it here (I don't know the answer) - but how much work is it to add pagination? I'm afraid someone will have a huge config and then we will need to modify this again (and cut a release, deploy and so on).

Reply:
OK, I overcame my laziness and implemented it the right way. 🙂